### PR TITLE
feat: Add option to excluded deprecated `PUT update` blueprint (default)

### PR DESCRIPTION
**Pull request by danielsharvey**
Sails 1.0 includes `PUT` and `PATCH` routes to the `update` blueprint although `PUT` deprecated; default to excluding the `PUT` route.

See
- https://sailsjs.com/documentation/reference/blueprint-api/update#?notes
- https://github.com/balderdashy/sails/blob/master/lib/hooks/blueprints/index.js#L401

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -85,6 +85,9 @@ module.exports['swagger-generator'] = {
             '500': { description: 'Internal server error' }
         }
     },
+    options: {
+      excludeDeprecatedPutBlueprintRoutes: true,
```
Comment (theoomoregbee) [inline README.MD:89]: just curious, any reason adding this to `options` what about just adding it directly to https://github.com/theoomoregbee/sails-hook-swagger-generator/blob/f60e61a8345f9dd9ad5c2c3cb319fb9eda251d35/lib/interfaces.ts#L5

Comment (theoomoregbee) [reply README.MD:89]: I feel it's a better ux to just add more options to `SwaggerGenerator`

Comment (danielsharvey) [reply README.MD:89]: Apologies, I added it this way and meant to ask you but forgot. I will change it as per your comment.

**Pull request merged**